modules/apps/remote-app/remote-app-admin-web/src/main/java/com/liferay/remote/app/admin/web/internal/deployer/RemoteAppEntryDeployerImpl.java

```diff
@@ -49,7 +56,8 @@
 			).put(
 				"com.liferay.portlet.instanceable", true
 			).put(
-				"javax.portlet.display-name", remoteAppEntry.getName()
+				"javax.portlet.display-name",
+				remoteAppEntry.getName(LocaleUtil.US)
```

Comment:
You can use LocaleUtil.getDefaultLocale or similar to avoid hardcoding one, but Brian will change it, so I wouldn't resent.

Reply:
Yeah, definitely, Brian said to hardcode it so he'd see it to eventually track this down.

Reply:
Thx @nhpatt yeah, I'll do that.